org.eclipse.scanning.sequencer/src/org/eclipse/scanning/sequencer/watchdog/TopupWatchdog.java

```diff
@@ -37,45 +38,98 @@
  * Will Rogers and Nick Battam are the ones who know and maintain this page.
  * If you are missing information on this page, please ask them to explain and possibly add information to the page.
  * The gruesome details of the top-up application are here: http://confluence.diamond.ac.uk/x/QpRTAQ 
- * 
- *
- <pre>
+
 SR-CS-FILL-01:COUNTDOWN: this is a float-valued counter that runs to zero
 at the start of TopUp and remains there until the fill is complete when 
 it resets to time before next TopUp fill,
 </pre>
  
- Example XML configuration
-    <pre>
+<h3> Example XML configuration</h3>

```

Comment:
Why is this Javadoc duplicated from above? It belongs in one or other but not both.